flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -551,91 +551,6 @@ public enum SchedulerType {
                                             code(SchedulerType.AdaptiveBatch.name()))
                                     .build());
 
-    @Documentation.Section({
```

Comment:
`JobManagerOptions` is `@PublicEnvlving` so we should only deprecate but not remove these options.
We can exclude them from the documentation with `@Documentation.ExcludeFromDocumentation`.